src/Testing/TypeInferenceTestCase.php

```diff
@@ -143,8 +145,10 @@ public function assertFileAsserts(
 	 */
 	public static function gatherAssertTypes(string $file): array
 	{
+		$pathHelper = new SimpleRelativePathHelper(dirname(__DIR__, 2));
```

Comment:
This doesn't make sense as it will be somewhere deep in the PHAR, for users that use TypeInferenceTestCase outside of phpstan-src.

Reply:
I committed my vision how it should have been done: https://github.com/phpstan/phpstan-src/pull/3166/commits/0e5311892b490f77e243ae2135d4a4e87126ff7b